src/plugins/english/mvlempyr.ts

```diff
@@ -1,22 +1,22 @@
 import { CheerioAPI, load as parseHTML } from 'cheerio';
 import { fetchApi } from '@libs/fetch';
 import { Plugin } from '@typings/plugin';
+import { Filters, FilterTypes } from '@libs/filterInputs';
 
 class MVLEMPYRPlugin implements Plugin.PluginBase {
   id = 'mvlempyr.com';
   name = 'MVLEMPYR';
   icon = 'src/en/mvlempyr/icon.png';
   site = 'https://www.mvlempyr.com/';
-  version = '1.0.0';
+  version = '1.0.1';
 
   headers = {
     'origin': 'https://www.mvlempyr.com/',
   };
 
-  _pageCache = new Map<string, string>();
-  _chapSite = 'https://chp.mvlempyr.com/';
-  _assetsSite = 'https://assets.mvlempyr.com/';
-  _allNovels: WPData[] | undefined;
+  _chapSite = 'https://chp.mvlempyr.net/';
```

Comment:
not fully sure what the difference between the .net and .com site are, but some novels dont work on .com :shrug: